notifications/src/main/java/org/openedx/notifications/presentation/inbox/NotificationsInboxViewModel.kt

```diff
@@ -150,4 +153,26 @@ class NotificationsInboxViewModel(
             )
         }
     }
+
+    fun navigateToPushNotificationsSettings(fm: FragmentManager) {
+        notificationsRouter.navigateToPushNotificationsSettings(fm)
+    }
+
+    fun markAllNotificationsAsRead() {
```

Comment:
Please place these functions in this order: 
1. Rest of the code
2. `markAllNotificationsAsRead`
3. `navigateToPushNotificationsSettings`
4. `emitErrorMessage`